src/AutoUI/schemaOps.ts

```diff
@@ -142,3 +143,30 @@ export const getFieldForFormat = (schema: JSONSchema, format: string) => {
 
 	return propertyKeyWithFormat;
 };
+
+export const convertRefSchemeToSchemaPath = (refScheme: string | undefined) => {
+	return !!refScheme
+		? refScheme
+				.split('.')
+				.join('.properties.')
+				.replace(/\[\d+\]/g, '.items')
+		: // TODO: This atm doesn't support ['my property']
+		  refScheme;
+};
+
+export const refSchemePrefix = (schema: JSONSchema) => {
```

Comment:
nit: `getRefSchemePrefix`? Seeing as you use `refSchemePrefix` as a parameter name elsewhere